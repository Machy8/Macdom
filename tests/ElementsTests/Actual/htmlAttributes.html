div id="id" class="class" data-label=label
div data-mix=;+=´´)ú¨§ů-.,-*/+_:\\()[]@&|\# id=mix class="class"
a href="{plink :Core:Front:Homepage:default" Link in on homepage in the Nette Framework
a n:href=":Core:Front:Homepage:default" Link in on homepage in the Nette Framework
input onclick='alert("Some text here 123 456 789")' onmouseover="alert('Some text here 123 456 789')"
@icon i .fa .fa-
icon pencil-square-o
