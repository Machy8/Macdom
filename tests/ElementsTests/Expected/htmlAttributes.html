<div class="class" id="id" data-label=label></div>
<div class="class" data-mix=;+=´´)ú¨§ů-.,-*/+_:\\()[]@&|\# id=mix></div>
<a href="{plink :Core:Front:Homepage:default">
	Link in on homepage in the Nette Framework
</a>
<a n:href=":Core:Front:Homepage:default">
	Link in on homepage in the Nette Framework
</a>
<input onclick='alert("Some text here 123 456 789")' onmouseover="alert('Some text here 123 456 789')">
